enterprise/backend/src/metabase_enterprise/content_translation/dictionary.clj

```diff
@@ -21,6 +21,12 @@
   [i]
   (+ i 2))
 
+(defn- collect-row-format-error
```

Comment:
this is wrong. The shape has changed so this is a map, always with 3 items so this will never catch this here. I added the check here: https://github.com/metabase/metabase/blob/content-translation/integration-branch/enterprise/backend/src/metabase_enterprise/content_translation/dictionary.clj#L68 it's already in the integration branch. Check this here when it's a tuple and then pass maps through that are all str/trimmed so we don't have to keep doing that.